_inc/client/plans-prompt/index.jsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Plans from '../plans';
+import { translate as __ } from 'i18n-calypso';
+import Gridicon from '../components/gridicon';
+
+export class PlansPrompt extends React.Component {
+	renderBanner() {
+		return (
+			<div className="plans-prompt__banner">
+				<div className="plans-prompt__logo">{ this.renderJetpackLogo() }</div>
+				<h2 className="plans-prompt__heading">{ __( 'Explore our Jetpack plans' ) }</h2>
+				<p className="plans-prompt__intro">
+					{ __( "Now that you're set up, pick a plan that fits your needs." ) }
+				</p>
+			</div>
+		);
+	}
+
+	renderJetpackLogo() {
```

Comment:
Should we move this into its own component, since we're already using something similar in `_inc/client/components/masthead/index.jsx`?

Reply:
Makes sense, yes 👍